resources/views/checkout/partials/sections/success/create-account.blade.php

```diff
@@ -2,11 +2,41 @@
     <x-rapidez-ct::title.lg>
         @lang('Create account')
     </x-rapidez-ct::title.lg>
-    <x-rapidez-ct::input
-        name="email"
-        type="email"
-        label="E-mailaddress"
-        v-bind:value="email"
-        disabled
-    />
+    <graphql-mutation
+        v-cloak
+        query="mutation customer ($firstname: String!, $lastname: String!, $email: String!, $password: String) { createCustomerV2 ( input: { firstname: $firstname, lastname: $lastname, email: $email, password: $password } ) { customer { email } } }"
+        redirect="/account"
+        :variables="{
+            firstname: order.customer_firstname,
+            lastname: order.customer_lastname,
+            email: order.customer_email
+        }"
+        :callback="registerCallback"
+    >
+        <form
+            class="mt-5 grid grid-cols-2 items-end gap-5"
+            slot-scope="{ mutate, variables, mutating }"
+            v-on:submit="mutate"
+        >
+            <x-rapidez-ct::input
+                name="email"
+                type="email"
+                label="E-mailaddress"
+                v-bind:value="order.customer_email"
+                disabled
+            />
+            <div class="text-sm text-ct-primary">
```

Comment:
is the text-ct-primary needed? probably not cause it's set as base font size color, can you check?